docs/user/dql/functions.rst

```diff
@@ -1219,15 +1219,60 @@ Example::
     +---------------------------+
 
 
+MAKEDATE
+--------
+
+Description
+>>>>>>>>>>>
+
+Returns a date, given `year` and `day-of-year` values. `dayofyear` must be greater than 0 or the result is `NULL`. The result is also `NULL` if either argument is `NULL`.
+Arguments are rounded to an integer.
```

Comment:
Was there a test case for this rounding behaviour in DateTimeFunctionTest?

Reply:
What about invalid positive numbers?  such as dayofyear `999`?

Reply:
> Was there a test case for this rounding behaviour in DateTimeFunctionTest?

Thanks, I will add.

> What about invalid positive numbers? such as dayofyear `999`?

There is no limit! Freedom!